cmd/serverless-init/cloudservice/cloudrun.go

```diff
@@ -64,3 +99,9 @@ func isCloudRunService() bool {
 	_, exists := os.LookupEnv(ServiceNameEnvVar)
 	return exists
 }
+
+func isCloudRunFunction() bool {
+	_, cloudRunFunctionMode := os.LookupEnv(functionTargetEnvVar)
+	_ = log.Warn(fmt.Sprintf("cloud function mode SET TO: %t", cloudRunFunctionMode))
```

Comment:
Does this really need to be assigned to `_` in order for it to work? I'm guessing no.

Also, I'm thinking this is more a debug level log than a warn. What are your thoughts on that?

Reply:
Great call out! I updated the log level to debug. Line 104 returns 2 variables to i need to keep the leading `_` as I am ignoring this return value